src/front-end/typescript/lib/pages/opportunity/sprint-with-us/edit/tab/history.tsx

```diff
@@ -61,8 +137,97 @@ const view: ComponentView<State, Msg> = ({ state, dispatch }) => {
   );
 };
 
+
+//props for the AttachmentsView component
+interface Props extends ComponentViewProps<State, Msg> {
+  disabled?: boolean;
+}
+
+// @duplicated-attachments-view
+const AttachmentsView: View<Props> = ({ state, dispatch, disabled }) => {
+  return (
+    <Row>
+      <Col xs='12'>
+        <Attachments.view
+          dispatch={mapComponentDispatch(dispatch, msg => adt('attachments' as const, msg))}
+          state={state.attachments}
+          disabled={disabled}
+          // className='mt-4'
+          accept={[".pdf"]}
+          />
+      </Col>
+    </Row>
+  );
+};
+
+
```

Comment:
I duplicated this section in CWU/SWU because State/Msg are specific to individual components